modules/private-clusters-setting-dns-private.adoc

```diff
@@ -4,13 +4,20 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="private-clusters-setting-dns-private_{context}"]
-= Setting DNS to private
+= Configuring DNS records to be published in a private zone
 
-After you deploy a cluster, you can modify its DNS to use only a private zone.
+For all {product-title} clusters, whether public or private, DNS records are published in a public zone by default.
+
+You can remove the public zone from the cluster DNS configuration to avoid exposing DNS records to the public. You may want to avoid exposing sensitive information, such as internal domain names, internal IP addresses, or the number of clusters at an organization, or you might simply have no need to publish records publicly. If all the clients that should be able to connect to services within the cluster use a private DNS service that has the DNS records from the private zone, then there is no need to have a public DNS record for the cluster.
```

Comment:
Per [ISG word usage](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#m), "may" should be reserved for indicating permission, and "might" should be used to indicate possibility:
```suggestion
You can remove the public zone from the cluster DNS configuration to avoid exposing DNS records to the public. You might want to avoid exposing sensitive information, such as internal domain names, internal IP addresses, or the number of clusters at an organization, or you might simply have no need to publish records publicly. If all the clients that should be able to connect to services within the cluster use a private DNS service that has the DNS records from the private zone, then there is no need to have a public DNS record for the cluster.
```